code/datums/spells/cluwne.dm

```diff
@@ -0,0 +1,88 @@
+/obj/effect/proc_holder/spell/targeted/touch/cluwne
+	name = "Curse of the Cluwne"
+	desc = "Turns the target into a fat and cursed monstrosity of a clown."
+	hand_path = /obj/item/weapon/melee/touch_attack/cluwne
+
+	school = "transmutation"
+
+	charge_max = 600
+	clothes_req = 1
+	cooldown_min = 200 //100 deciseconds reduction per rank
+
+	action_icon_state = "clown"
+
+/mob/living/carbon/human/proc/makeCluwne()
+	to_chat(src, "<span class='danger'>You feel funny.</span>")
+	adjustBrainLoss(80)
+	nutrition = 9000
+	overeatduration = 9000
+	confused = 120
```

Comment:
Actually, that's not correct, it's not permanent.

`change_misstep_chance` directly alters `misstep_chance`:  [[source]](https://github.com/goonstation/goonstation-2016/blob/d8a2d60915fd3b74653a1b7d4b8a0910c6fc2f19/code/mob/living.dm#L1057).

And `misstep_chance` ticks down every `Life` like confusion does: [[source]](https://github.com/goonstation/goonstation-2016/blob/d8a2d60915fd3b74653a1b7d4b8a0910c6fc2f19/code/mob/living/carbon/human.dm#L3655)